deploy/lambda/whitelist.py

```diff
@@ -0,0 +1,80 @@
+import json
+import boto3
+import botocore
+from botocore.vendored import requests
+
+# Config global var before deployment
+REMOVE_PORTS = []
+WHITELIST_SERVICE = 'https://my.service.org/whitelist'
+
+
+def get_nodes_ips():
+    data = requests.get(url=WHITELIST_SERVICE)
+    return json.loads(data.json()['body'])
+
+
+def get_ip_perm_module(ip):
+    return {
+        'IpProtocol': "tcp",
+        'FromPort': 11625,
+        'ToPort': 11625,
+        'IpRanges': [{'CidrIp': ip, 'Description': f'Node - {ip}'}]
+    }
+
+
+def generate_ip_permissions():
+    ip_permissions = [get_ip_perm_module(ip) for ip in get_nodes_ips()]
+    return ip_permissions
+
+
+def remove_rules_from_sg(security_group):
+    try:
+        rules_to_delete = []
+        for rule in security_group.ip_permissions:
+            from_port = rule.get('FromPort')
+            to_port = rule.get('ToPort')
+            if from_port in REMOVE_PORTS or to_port in REMOVE_PORTS:
+                rules_to_delete.append(rule)
+        security_group.revoke_ingress(IpPermissions=rules_to_delete) if rules_to_delete else None
```

Comment:
i don't understand what `if rules_to_delete else None` does here. keep it simple and readable for others (and also future you a year from now):
```python
if rules_to_delete:
    security_group.revoke_ingress(IpPermissions=rules_to_delete)
```